.eslintrc.js

```diff
@@ -24,7 +24,6 @@ module.exports = {
 
         'no-duplicate-imports': 'error',
         'no-console': 'error',
-        'no-magic-numbers': ['error', { 'ignoreArrayIndexes': true }],
```

Comment:
more annoying than with it for now, got an error when checking the length of an array being greater than 1. No real sense in adding a named variable for this.